code/go/0chain.net/smartcontract/dbs/event/writemarker.go

```diff
@@ -41,6 +41,16 @@ func (edb *EventDb) GetWriteMarker(txnID string) (*WriteMarker, error) {
 	return &wm, nil
 }
 
+func (edb *EventDb) GetDataWrittenToAllocationForLastNBlocks(blockNumber int64) (int64, error) {
```

Comment:
Rename to GetAllocationWrittenSizeInLastNBlocks()

Reply:
done